src/js/page.js

```diff
@@ -336,14 +338,44 @@ export default {
     },
     _getTocHtmlFragment( tocItems ) {
         const items = document.createDocumentFragment();
+        let currentGroupList;
         tocItems.forEach( item => {
+            const $itemPageEl = $( item.pageEl );
```

Comment:
let's not use jQuery. We're in the process of removing it, and it has no advantage here.